docs/configuration.md

```diff
@@ -0,0 +1,45 @@
+# Configuration
+LebkuchenFM uses environment variables for configuration.
+
+You can also create `.env` file in the root of this project and put desired configuration variables in it.
```

Comment:
It is not so straightforward now. Backend service does not read this automatically. Maybe we should describe this with recommended IDE as an example. Or even describe `.env` files as recommended solution but with "some caveats" - .env plugin in intelij, docker compose env file vs plain properties in yaml? 🤔

Reply:
That's very true. I totally forgot about it.